### DISCUSSION TITLE
Issue on Mac finder sidebar icon (MacOS 14.3.1)

### DISCUSSION THREAD

**Discussion by zenglinx** [iterate-ch/cyberduck#15663] [Q&A] [answered]
On my macbook(14.3.1), mountainduck(4.15.3), the finder sidebar icon of webdav driver is wrong, which is equivalent to the application icon, pls look at the following picture:
<img width="1219" alt="image" src="https://github.com/iterate-ch/cyberduck/assets/9171954/4a423fff-c5ee-4e76-bf8c-c5cc8c6d068e">

Attached some information of mountainduck:
<img width="707" alt="image" src="https://github.com/iterate-ch/cyberduck/assets/9171954/30852cbd-8159-4a37-ac1b-b5929dd281ee">
<img width="295" alt="截屏2024-02-28 11 12 07" src="https://github.com/iterate-ch/cyberduck/assets/9171954/7b01cdf8-0fc8-4f6e-852f-91cf678240b9">

**Comment by dkocher** [comment, marked as answer]
The _About_ screen suggests you have replaced the application icon of the `.app` package. The original icon includes the icon to be displayed in the sidebar.

**Reply by zenglinx** [reply]
Thank you for your comment. After recovering the application icon, the issue has been resolved.

**Comment by zenglinx** [comment]
If you need to change the icon of Mountain Duck, the process can be a bit complex. Apart from modifying the application.icns file, you will also need to update the Assets.car file.

Here is the step-by-step process:
1. Utilize the `iconutil` command tool to change the icons within the application.icns file.
2. Use the [Samra](https://github.com/NSAntoine/Samra) application to edit the icons within the Assets.car file, which deals with Asset Catalog (.car) files.
